com.unity.template.renderstreaming-hd/Assets/Scripts/Editor/RenderStreamingEditor.cs

```diff
@@ -1,6 +1,8 @@
 using Unity.RenderStreaming;
 using UnityEditor;
 using UnityEngine;
+using Unity.RenderStreaming.Signaling;
+using Boo.Lang;
```

Comment:
I am not sure why it is needed.

Reply:
I will check.

To be honest I don't really know where the Boo.Lang comes from ?!! (perhaps another effect of the auto merge magic after 6 months of separated work)

Reply:
Oh it seems it came with List<>, strange I'll found a more common using

Reply:
Fixed, I changed this strange auto-using to the common System.Collections.Generic.
Unity.RenderStreaming.Signaling is used for finding classes in the project that implements ISignaling